### PR TITLE
This completes the linguistic review for grpc_unary_blocking repository.

### PR DESCRIPTION
**Pull request by RukshiW**
## Documentation
Commit 1: Completes the linguistic review for grpc_unary_blocking.

### CODE REVIEW

**Comment by RukshiW** [comment]
Commit 2: This completes the linguistic review for grpc_bidirectional_streaming repository.

File: examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.bal [hunk 1]
```diff
@@ -17,13 +17,13 @@ endpoint grpc:Listener ep {
 service<grpc:Service> Chat bind ep {
     map<grpc:Listener> consMap;
 
-    //This resource is triggered when a new service is initializing
+    //This resource is triggered when a new service is initialized.
```
Comment (daneshk) [inline examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.bal]: shall we make it like "new caller connection is initialized"

File: examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.bal [hunk 2]
```diff
@@ -17,13 +17,13 @@ endpoint grpc:Listener ep {
...
     onOpen(endpoint caller) {
         var connID = caller.id;
         consMap[<string>connID] = caller;
     }
 
-    //This resource is triggered when client need to send request to service
+    //This resource is triggered when the client needs to send a request to the service.
```
Comment (daneshk) [inline examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.bal]: shall we make it like "when caller send request message to the service"

File: examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.bal [hunk 3]
```diff
@@ -39,14 +39,14 @@ service<grpc:Service> Chat bind ep {
         }
     }
 
-    //This resource is triggered when some error has been happen at server end
+    //This resource is triggered when an error has occurred at the server end.
```
Comment (daneshk) [inline examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.bal]: shall we make it like "when server receives error message from the caller"

File: examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.bal [hunk 4]
```diff
@@ -39,14 +39,14 @@ service<grpc:Service> Chat bind ep {
...
     onError(endpoint caller, error err) {
         if (err != ()) {
             io:println("Something unexpected happens at server : " + err.message);
         }
     }
 
-    //This resource is triggered when client notify service that it has finish send requests
+    //This resource is triggered when the client notifies the service that it has finished sending requests.
```
Comment (daneshk) [inline examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.bal]: shall we make it like "when server receives complete notification from the caller. Caller sends complete notification when it has finished sending messages"

**Review by daneshk — APPROVED** [review]

**Pull request merged**